.gitlab-ci.yml

```diff
@@ -63,18 +38,35 @@ before_script:
   CI_PLATFORM: "do"
   SSH_USER: root
 
+.packet_variables: &packet_variables
+  CI_PLATFORM: "packet"
+  SSH_USER: kubespray
 
-.testcases: &testcases
-  <<: *job
-  <<: *docker_service
+.packet-job: &packet-job
+  tags:
+    - packet
+  image: quay.io/kubespray/kubespray:v2.8.1
+
+.gce-job: &gce-job
+  tags:
+    - kubernetes
+    - docker
+  image: quay.io/kubespray/kubespray:v2.8.1
+
+.docker_service: &docker_service
+  services:
+     - docker:dind
+
+.cache: &cache
   cache:
     key: "$CI_BUILD_REF_NAME"
     paths:
       - downloads/
       - $HOME/.cache
+
+.testcases: &testcases
   before_script:
-    - docker info
```

Comment:
what do you mean it's not used ? it is you can check the logs from the jobs in the pipeline

Reply:
docker info inside a vm with no docker is not working

Reply:
I'm not sure to understand, gce are VM

Reply:
You cannot get docket info before docker is installed